.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
-      options: -v /tmp:/tmp # make sure Claicore blob can access tmp folder
+      volumes: /tmp:/tmp
```

Comment:
It needs to be a list. This will fail in the linter.

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainervolumes

Reply:
Yeah, I just saw the error message